api/organisations/models.py

```diff
@@ -125,6 +131,11 @@ def update_plan(self, plan_id):
         self.max_seats = get_max_seats_for_plan(plan_id)
         self.save()
 
+    @hook(AFTER_CREATE)
+    @hook(AFTER_SAVE, when="cancellation_date", was_not=None, is_now=None)
+    def subscribe_all_users_to_mailing_list(self):
```

Comment:
This confused me for a little while but I think I understand now that it's more of an update to make sure all subscribed users are listed as paid? 

Should we have a similar hook when a subscription is cancelled?

Reply:
Yeah, it's when an org bought the subscription then cancelled it and then finally decided to buy one again.

Reply:
> Should we have a similar hook when a subscription is cancelled?

To mark all the users as not paid? Maybe in the next release?

Reply:
I think we could just use the same hook tbh and then base the decision on what the new cancellation date is?  

```python
@hook(AFTER_SAVE, when="cancellation_date", has_changed=True)
```

Reply:
that will not trigger the hook for create since cancellation_date did not change after initializing the model